recipe/test/test_qmimedatabase.cpp

```diff
@@ -0,0 +1,8 @@
+#include <QMimeDatabase>
+#include <cassert>
+
+
+int main(int argc, char *argv[]) {
```

Comment:
Is this test being executed? I don't see any changes on https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/run_test.sh nor https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/run_test.bat

Reply:
> Is this test being executed? I don't see any changes on https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/run_test.sh nor https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/run_test.bat


It is executing now.